### ISSUE TITLE
No code hints for standard library objects

### ISSUE THREAD

**Issue by gaschlabgit** [spyder-ide/spyder#2640]
## Description

**What steps will reproduce the problem?**

1. Anytime I am using spyder with standard libs, code hint do not show argument information
2. 
3. 

**What is the expected output? What do you see instead?**
I see a little yellow box, that says arguments, with empty brackets

**Please provide any additional information below**

## Version and main components

* Spyder Version: 2.3.5.2 
* Python Version: 3.4.3
* Qt Versions:  4.8.6, PyQt4 (API v2) 4.11.3 on Linux

## Optional dependencies
```
pyflakes >=0.6.0:  0.9.2 (OK)
pep8 >=0.6      :  1.6.2 (OK)
IPython >=1.0   :  3.2.0 (OK)
zmq >=2.1.11    :  14.7.0 (OK)
pygments >=1.6  :  2.0.2 (OK)
pandas >=0.13.1 :  0.16.2 (OK)
sphinx >=0.6.6  :  1.3.1 (OK)
psutil >=0.3    :  2.2.1 (OK)
rope >=0.9.2    :  0.9.4-1 (OK)
jedi >=0.8.1    :  0.8.1 (OK)
matplotlib >=1.0:  1.4.3 (OK)
sympy >=0.7.3   :  0.7.6 (OK)
pylint >=0.25   :  None (NOK)

```

**Comment by ccordoba12** [comment]
Could you follow the advice in this comment: https://github.com/spyder-ide/spyder/issues/986#issuecomment-98342587 to see if that fixes your problem?

**Comment by ccordoba12** [comment]
Closing because of lack of response.

**Comment by gaschlabgit** [comment]
Yes, that fixed it thank you

**Comment by ccordoba12** [comment]
I decided to fix this problem for good in Spyder 2.3.7 by not allowing users to change the Qt theme in Ubuntu.
